src/sql/magic_cmd.py

```diff
@@ -35,15 +35,39 @@ def error(self, message):
 class SqlCmdMagic(Magics, Configurable):
     """%sqlcmd magic"""
 
+
     @line_magic("sqlcmd")
     @magic_arguments()
     @argument("line", default="", type=str, help="Command name")
-    def execute(self, line="", cell=""):
+    def _validate_inputs(self, line=""):
+        """
+        Command
+        """
+
+        if line == "":
+            raise UsageError(
+                "Missing argument for %sqlcmd. "
+                "Valid commands are: 'tables', 'columns', 'profile'"
```

Comment:
So you can utilize the `AVAILABLE_SQLCMD_COMMANDS` too, something like

```suggestion
                "Valid commands are: {}".format(','.join(AVAILABLE_SQLCMD_COMMANDS))
```